x-pack/plugin/esql/qa/testFixtures/src/main/resources/math.csv-spec

```diff
@@ -203,7 +203,7 @@ height:double | s:double
 powSalarySquared
 from employees | eval s = pow(to_long(salary) - 75000, 2) + 10000 | keep salary, s | sort salary desc | limit 4;
 
-salary:integer | s:long
+salary:integer | s:double
```

Comment:
@not-napoleon, I have merged the mixed-cluster tests, you might need to add #[skip:-8.11.99] to these tests, as the assertions differ on 8.11.